binary_transparency/firmware/devices/usb_armory/README.md

```diff
@@ -140,11 +141,31 @@ There are some invariants which must hold for this chain to work:
     following contents:
     * `armory-boot.conf` - a JSON file which tells the bootloader which files
       to load
-    * a valid ARM linux Kernel image
-    * a valid DTB file
-   Note that the `armory-boot.conf` file also contains SHA256 hashes of the
-   kernel and DTB files, and these MUST be correct.
+    * Either:
+        * to boot a Linux Kernel:
+           * a valid ARM linux Kernel image
+           * a valid DTB file
+        * to boot ELF unikernel: 
+           * a valid ARM bare-metal ELF binary/unikernel
 
+   Note that the `armory-boot.conf` file also contains SHA256 hashes of 
+   all files referenced, and these MUST be correct.A
+
+To aid in the creation of valid firmware images, the 
```

Comment:
better to say at the end of line 154, use the following script

Reply:
done